user-guide/Troubleshooting/Procedures/Troubleshooting_Cassandra.md

```diff
@@ -0,0 +1,157 @@
+---
+uid: Troubleshooting_Cassandra
+---
+
+# Troubleshooting – Cassandra
+
+## Context
```

Comment:
Is this "Context" title necessary? Usually, when we add context at the top of a page, we do so directly under the main title, so keeping this title will clash a bit with the rest of the documentation.

Reply:
It's OK to remove the Context sub title.